cmd/list.go

```diff
@@ -54,6 +55,7 @@ const (
 	blobAccessTier   validProperty = "BlobAccessTier"
 	contentType      validProperty = "ContentType"
 	contentEncoding  validProperty = "ContentEncoding"
+	contentMD5		         validProperty = "md5"
```

Comment:
This name does not align with the helptext. Otherwise, looks good.

Reply:
Also, formatting here.

Reply:
Fixed